continuum/task_set.py

```diff
@@ -109,19 +150,16 @@ def split_train_val(dataset: TaskSet, val_split: float = 0.1) -> Tuple[TaskSet,
     :return: A tuple a dataset, respectively for train and validation.
     """
     random_state = np.random.RandomState(seed=1)
-
-    indexes = np.arange(len(dataset.x))
+    indexes = np.arange(len(dataset))
     random_state.shuffle(indexes)
 
     train_indexes = indexes[int(val_split * len(indexes)):]
     val_indexes = indexes[:int(val_split * len(indexes))]
 
-    x, y, t = dataset.x, dataset.y, dataset.t
-    train_dataset = TaskSet(
-        x[train_indexes], y[train_indexes], t[train_indexes], dataset.trsf, dataset.data_type
-    )
-    val_dataset = TaskSet(
-        x[val_indexes], y[val_indexes], t[val_indexes], dataset.trsf, dataset.data_type
-    )
+    x_train, y_train, t_train = dataset.get_raw_samples_from_ind(train_indexes)
```

Comment:
Why not using directly `x` `y` `z`?

As long as we are in internal code, no need to wrap getter.

Reply:
Is this comment it still up to date?
In the current version, I used special name to differentiate train and val data